readthedocs/core/static-src/core/js/doc-embed/rtd-data.js

```diff
@@ -40,8 +40,9 @@ var configMethods = {
 
     show_promo: function () {
         return (
-            this.api_host !== 'https://readthedocs.com' &&
-            this.theme_supports_promo());
+            this.api_host === 'https://readthedocs.org' ||
+            this.api_host === 'http://127.0.0.1:8000'
```

Comment:
Should this also check for `localhost`? Also feels a bit hacky, but I don't know a better way.

Reply:
The default in `settings/base.py` is `127.0.0.1:8000`. It is a bit hacky but the above change covers the defaults.